charts/osm/crds/multiclusterservice.yaml

```diff
@@ -0,0 +1,49 @@
+{{ if .Values.enableMulticlusterMode }}
```

Comment:
CRDs under `charts/osm/crds` cannot be templated: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you